lib/osf-components/addon/components/search-page/styles.scss

```diff
@@ -39,6 +39,71 @@
     }
 }
 
+.heading-wrapper-institutions,
+.heading-wrapper-institutions-mobile {
+    background-color: $osf-dark-blue-navbar;
+    color: $color-text-white;
```

Comment:
Yes, we met to discuss this. Our default blue has the possibility of being mistaken for another institution's blue, which is something we would like to avoid. By specifying this here, it prevents that design conflict of interest.